### ISSUE TITLE
Missing class definition when running unit tests.

### ISSUE THREAD

**Issue by EBatTiVo** [TiVo/intellij-haxe#69]
Test: HaxeFindUsagesTest

Occurs on JetBrains and TiVo repos.

The following exception is thrown when running the Haxe plugin unit tests:
<pre>
java.lang.NoClassDefFoundError: io/netty/channel/EventLoopGroup
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:249)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:408)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:398)
	at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.access$000(ComponentManagerImpl.java:384)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:107)
	at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:89)
	at com.intellij.openapi.components.impl.stores.ApplicationStoreImpl.load(ApplicationStoreImpl.java:87)
	at com.intellij.openapi.application.impl.ApplicationImpl.load(ApplicationImpl.java:508)
	at com.intellij.idea.IdeaTestApplication$1.run(IdeaTestApplication.java:54)
	at com.intellij.openapi.application.RunResult.run(RunResult.java:38)
	at com.intellij.openapi.application.WriteAction$1$1.run(WriteAction.java:47)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:984)
	at com.intellij.openapi.application.WriteAction$1.run(WriteAction.java:44)
	at com.intellij.openapi.application.WriteAction.execute(WriteAction.java:53)
	at com.intellij.idea.IdeaTestApplication.getInstance(IdeaTestApplication.java:52)
	at com.intellij.testFramework.fixtures.impl.HeavyIdeaTestFixtureImpl.b(HeavyIdeaTestFixtureImpl.java:176)
	at com.intellij.testFramework.fixtures.impl.HeavyIdeaTestFixtureImpl.setUp(HeavyIdeaTestFixtureImpl.java:101)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl$18.run(CodeInsightTestFixtureImpl.java:1123)
	at com.intellij.util.ui.UIUtil.invokeAndWaitIfNeeded(UIUtil.java:2074)
	at com.intellij.testFramework.fixtures.impl.CodeInsightTestFixtureImpl.setUp(CodeInsightTestFixtureImpl.java:1119)
	at com.intellij.testFramework.fixtures.JavaCodeInsightFixtureTestCase.setUp(JavaCodeInsightFixtureTestCase.java:57)
	at com.intellij.testFramework.UsefulTestCase.defaultRunBare(UsefulTestCase.java:340)
	at com.intellij.testFramework.UsefulTestCase$3.run(UsefulTestCase.java:353)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:199)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:672)
	at java.awt.EventQueue.access$400(EventQueue.java:81)
	at java.awt.EventQueue$2.run(EventQueue.java:633)
	at java.awt.EventQueue$2.run(EventQueue.java:631)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:642)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: java.lang.ClassNotFoundException: io.netty.channel.EventLoopGroup
	at java.net.URLClassLoader$1.run(URLClassLoader.java:202)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:301)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
	... 39 more
</pre>

TIR: STB-5630

**Comment by EBatTiVo** [comment]
I've finally tried this and can confirm that the exceptions go away when the netty library is included in the SDK. (For me, I'm using idea-UI-135.1286, so the version I needed was ${intellij_install_directory}/lib/netty-all-4.1.0.Alpha1.jar).

I've updated the README.md file to state the above, along with some other documentation that I've been putting off writing.  This issue can be closed as soon as I integrate it back to github.
